app/src/main/java/org/mifos/mobile/api/services/LoanAccountsListService.java

```diff
@@ -17,18 +17,21 @@
 import retrofit2.http.Path;
 import retrofit2.http.Query;
 
+import static org.mifos.mobile.utils.Constants.LOAN_ID;
+
 /**
  * @author Vishwajeet
  * @since 23/6/16.
  */
 public interface LoanAccountsListService {
 
+
     @GET(ApiEndPoints.LOANS + "/{loanId}/")
-    Observable<LoanAccount> getLoanAccountsDetail(@Path("loanId") long loanId);
+    Observable<LoanAccount> getLoanAccountsDetail(@Path(LOAN_ID) long loanId);
```

Comment:
I don't think this is necessary. API params should not be in constants. Even if they are, then you will also need to use the same constant in `@GET(ApiEndPoints.LOANS + "/{loanId}/")` then only it makes sense

Reply:
actually I was getting a pmd violation when simply using "loanId" stating "too much use of this and shoudl be converted to a variable/constant"
So what should I do now?

Reply:
okay, fine, let's keep it as is then